One `OrderedSet` test fails

OS: Linux, Ubuntu 16.04

Commands ran:

```console
felix@felix-X550LD ~/D/P/A/commandant> rm -rf .build/
felix@felix-X550LD ~/D/P/A/commandant> git checkout 0.13.0 
Note: checking out '0.13.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 7f29606... Release 0.13.0
felix@felix-X550LD ~/D/P/A/commandant> swift test
```

Failed test output:

```console
Test Case 'XCTestCase.OrderedSet, should preserve the order of the given input' started at 2018-04-09 22:11:17.477
/home/felix/Documents/Programming/AwesomeSystemSwift/commandant/Tests/CommandantTests/OrderedSetSpec.swift:17: error: XCTestCase.OrderedSet, should preserve the order of the given input : failed - expected to not equal <acb>, got <acb>

Test Case 'XCTestCase.OrderedSet, should preserve the order of the given input' failed (0.0 seconds)
```

What's the output on that line under Mac OS? Maybe it's failing on both ends.

--- a/Tests/CommandantTests/OrderedSetSpec.swift
+++ b/Tests/CommandantTests/OrderedSetSpec.swift
@@ -14,7 +14,6 @@ class OrderedSetSpec: QuickSpec {
 
 			it("should preserve the order of the given input") {
 				let input = ["a", "c", "b", "a"]
-				expect(Set(input).joined()) != "acb"
 				expect(OrderedSet(input).joined()) == "acb"
 			}
 		}

